### PR TITLE
Merging My feature

### PR DESCRIPTION
**Pull request by markgrebe**
I'm merging the feature and I want everyone to review.

### CODE REVIEW

File: src/main/js/components/EditComponent.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            make: '',
+            model: '',
+            year: '',
+        }
+        this.saveCar = this.saveCar.bind(this);
+        this.loadCar = this.loadCar.bind(this);
+    }
+
+    componentDidMount() {
+        this.loadCar();
```
Comment (mgrebe-cu) [inline src/main/js/components/EditComponent.js:19]: Should this be this.loadMyCar()

File: src/main/js/components/EditComponent.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    onChange = (e) =>
+        this.setState({ [e.target.name]: e.target.value });
+
+    saveCar = (e) => {
+        e.preventDefault();
+        let car = {id: this.state.id, make: this.state.make, model: this.state.model, year: this.state.year};
+        ApiService.editCar(car)
+            .then(res => {
+                this.setState({message : 'Car added successfully.'});
+                this.props.history.push('/');
```
Comment (mgrebe-cu) [inline src/main/js/components/EditComponent.js:44]: Should there be white space here.

**Review by mgrebe-cu — CHANGES_REQUESTED** [review]

File: src/main/js/components/EditComponent.js (hunk 1 above)
Comment (markgrebe) [reply src/main/js/components/EditComponent.js:19]: No, it shouldn't.

File: src/main/js/components/EditComponent.js (hunk 2 above)
Comment (markgrebe) [reply src/main/js/components/EditComponent.js:44]: How much white space?

Comment (mgrebe-cu) [reply src/main/js/components/EditComponent.js:44]: 3 spaces.

Comment (markgrebe) [reply src/main/js/components/EditComponent.js:44]: OK

**Pull request closed without merging**